### PR TITLE
task(UIS-55): Delete Inventory Item Entity

### PR DESCRIPTION
**Pull request by Kam1st**
JIRA: https://champlainsaintlambert.atlassian.net/browse/UIS-55

- delete inventory item methods, dto's, classes
- optimized imports everywhere

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,13 +1,5 @@
 services:
 
-#  janus:
```
Comment (StaceyJin) [inline docker-compose.yml:3]: What component was nginx supporting again? I'm assuming removal won't affect Docker performance?

File: src/main/java/com/um/inventoryservice/PresentationLayer/OrderController.java [hunk 2]
```diff
@@ -2,7 +2,6 @@
 
 import com.um.inventoryservice.BusinessLayer.OrderService;
 import com.um.inventoryservice.DataLayer.OrderDTO;
-import com.um.inventoryservice.DataLayer.StockItemDTO;
```
Comment (StaceyJin) [inline src/main/java/com/um/inventoryservice/PresentationLayer/OrderController.java:5]: A lot of StockItemDTO removals as well, just curious as to why.

**Review by StaceyJin — APPROVED** [review]
Changes made all appear logical and in order.

File: docker-compose.yml (hunk 1 above)
Comment (Kam1st) [reply docker-compose.yml:3]: Wasn't using anything, it was something we tried to help resolve docker issues but it's not needed.

File: src/main/java/com/um/inventoryservice/PresentationLayer/OrderController.java (hunk 2 above)
Comment (Kam1st) [reply src/main/java/com/um/inventoryservice/PresentationLayer/OrderController.java:5]: Not using them in the classes, must've been imported when copy/pasted some code.

**Pull request merged**